spec/controllers/recommended_links_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'spec_helper'
+
+describe RecommendedLinksController do
+  let(:recommended_link_params) { { title: 'Tax', link: 'https://www.tax.service.gov.uk/', description: 'Self assessment', keywords: 'self, assessment, tax', search_index: 'mainstream' } }
```

Comment:
Let's multi-line this, with a `do...end` block and appropriate formatting of the hash.